trace/opencensus/trace/ext/django/config.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib
+
+from django.conf import settings as django_settings
+
+DEFAULT_DJANGO_TRACER_CONFIG = {
+    'TRACER': 'opencensus.trace.tracer.context_tracer.ContextTracer',
+    'SAMPLER': 'opencensus.trace.samplers.always_on.AlwaysOnSampler',
+    'REPORTER': 'opencensus.trace.reporters.print_reporter.PrintReporter'
+}
+
+PATH_DELIMITER = '.'
+
+
+class DjangoTraceSettings(object):
+    """Set the params for a django tracer, including the tracer, sampler and
+    reporter. If the user did not define the settings in django settings file.
+    then use the dafaults.
+    """
+
+    def __init__(self, settings=DEFAULT_DJANGO_TRACER_CONFIG):
+        # Try to read the user settings from django settings file
+        user_settings = getattr(django_settings, 'OPENCENSUS_TRACE', None)
+
+        if user_settings is not None:
```

Comment:
Could simplify this to `self.settings = getattr(django_settings, 'OPENCENSUS_TRACE', user_settings)`